security/rbac/scripts/enable-rbac-connect.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+
+################################################################################
+# Overview
+################################################################################
+#
+################################################################################
+
+# Source library
+. ../../../utils/helper.sh
+. ./rbac_lib.sh
+
+check_env || exit 1
+check_cli_v2 || exit 1
+check_jq || exit 1
+
+##################################################
+# Initialize
+##################################################
+
+. ../config/local-demo.cfg
+ORIGINAL_CONFIGS_DIR=/tmp/original_configs
+DELTA_CONFIGS_DIR=../delta_configs
+FILENAME=connect-avro-distributed.properties
+create_temp_configs $CONFLUENT_HOME/etc/schema-registry/$FILENAME $ORIGINAL_CONFIGS_DIR/$FILENAME $DELTA_CONFIGS_DIR/${FILENAME}.delta
+
+# Log in to Metadata Server (MDS)
+login_mds $MDS
+
+##################################################
+# Administrative Functions
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:connect-configs
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:connect-offsets
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:connect-statuses
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Group:connect-cluster
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:_secrets (for Secret Registry)
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Group:secret-registry (for Secret Registry)
+##################################################
+
+# Get the Kafka cluster id
+get_cluster_id_kafka
+
+for CONNECT_TOPIC in connect-configs connect-offsets connect-statuses; do
+  echo -e "\n# Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:${CONNECT_TOPIC}"
```

Comment:
Nit: I think the language here can be tweaked. My understanding is that a [principal](https://docs.oracle.com/javase/8/docs/api/java/security/Principal.html) (at least in the Java ecosystem) is an identity like a user or a service account, so I think it might make more sense to word this like "Grant the principal <principal> the role <role> for the resource <resource>" or "Grant the principal <principal> the role <role> for the cluster <cluster>".

Reply:
@C0urante thanks for bringing this up.  I'll pursue this separately, and it will apply to all components, not just connect

Reply:
Resolved